redwood-treehouse-host/src/commonTest/kotlin/app/cash/redwood/treehouse/FakeDispatchers.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.treehouse
+
+import kotlinx.coroutines.CoroutineDispatcher
+
+class FakeDispatchers(
+  override val ui: CoroutineDispatcher,
+  override val zipline: CoroutineDispatcher,
```

Comment:
As we're not using these values, perhaps remove the constructor and just override these values in the class body?

Reply:
This one’s tricky. They’re `override` values from the interface.